db/seeds/02_items.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO items (
+  name, description, price, image_url
+) VALUES (
+  'Classic', 'Signature bubble tea with 3 toppings: bubbles, pudding and grass jelly', 6.99, 'https://images.squarespace-cdn.com/content/v1/5edfb8e7559cd9129deae815/1629474098758-PX9OU33P0MIPXHMDD4JL/2+Guys.png'
```

Comment:
I'll look for the product images and create an images folder in public. The path will be "/images/image_name". Not sure if it's good to use pics with other logo on it lol

Reply:
Yup. I just put something in there for sample data.

Reply:
Nice pics!